e2e/interchaintestv8/relayer/config.tmpl

```diff
@@ -1,6 +1,6 @@
 {
   "server": {
-    "log_level": "debug",
+    "log_level": "info",
```

Comment:
Fair. It would be nice to be able to override this with an env variable as it can be quite useful when debugging. But we can maybe that at some later point :)